packages/FlexLoader/tests/Flex/FlexLoader/Source/MicroKernelTraitKernel.php

```diff
@@ -52,8 +52,6 @@ protected function configureRoutes(RouteCollectionBuilder $routeCollectionBuilde
 
     protected function configureContainer(ContainerBuilder $containerBuilder, LoaderInterface $loader): void
     {
-        $loader->load(__DIR__ . '/project-dir-micro-kernel/config/main-config.yaml');
```

Comment:
Why this change?

Reply:
as I moved main-config.yaml to parameters.yaml where you would expected parameters like  kernel.secret to be present, then its loaded by FlexLoader by design as it is the main goal of this PR, so you dont need to load it manually

Reply:
I see